test/test-parser.js

```diff
@@ -1080,8 +1381,10 @@ describe('html error formatting', () => {
 });
 
 describe('validate', () => {
+  // For simple validation purposes — a noop “onToken” can be provided.
+  const onToken = () => {};
   // eslint-disable-next-line no-shadow
-  const html = strings => wrapper(validator, strings);
+  const html = strings => XParser.parse(strings, onToken);
```

Comment:
It’s worth calling out how _simple_ the logic to create a “validator” becomes here.